### DISCUSSION TITLE
Lesson 7 - I can't use the test or script like Patricks example

### DISCUSSION THREAD

**Discussion by Loupsteb** [Cyfrin/foundry-full-course-cu#1311] [Q&A]
hi,
j'ai commencé depuis peu le cours de Patrick sur Foundry, j'ai récupéré le code fundMe qui met a notre disposition mais cependant lorsque j'essaie de reproduire les example qui met en evidence dans la vidéo a chaque fois on me demande de renseigner un parametre.
Je pense que ca vient du code fundMe a utiliser, au début j'ai utilisé le code de la lecon 4 et ca me bloquait, là j'essaie avec le code fundMe renseigner en lecon 7 et j'ai le meme probleme.
Pourriez-vous m'indiquer quel code fundMe je dois utiliser pour suivre au mieux so
![image_2024-01-10_162736506](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/6d12fd0a-0ea5-45c8-b4c0-4e347e41ebad)
n cours ?
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/d849a532-ae3d-4833-8bfb-f87507fd53a5)
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/52604984/56505478-af91-4bfb-a30f-17dfbd53d45a)

**Comment by vascofcd** [comment]
Hey there! It'd be super helpful if you could post your questions in English – makes it easier for everyone to understand (it would be nice if you could translate your question, people may come across the same problem as you). Also, looks like you're using the FundMe code from lesson 7. You should actually be using the one from lesson 4, the Remix project. Check out the right code here: https://github.com/Cyfrin/remix-fund-me-f23. Hope that helps!
